wao/util.py

```diff
@@ -0,0 +1,70 @@
+EXECUTION_PATH = '/root/workspace2/execution'  # do not move this to config
+from wao.config import Config
+import sys
+import time
+import watchdog
+import threading
```

Comment:
this file already exists in the repo - something is wrong in the conflict resolution - create a new pr